### DISCUSSION TITLE
Lotus block sync crash with windowVerifyError and just stop synchronize....

### DISCUSSION THREAD

**Discussion by lionsoul2014** [filecoin-project/lotus#7846] [Build and Sync] [answered]
**Version (run `lotus version`):**
```
0.5.4 and 0.5.6
```

**Describe the bug**
Well, it works find until today and i have to restart it often to keep going.
Every time i restart it and the miner will got lots of sectors Marked as PreCommitFailed or CommitFailed and cannot be recoved.
Check the the following screenshots for more details

**Screenshots**
![2020-08-30_18-10](https://user-images.githubusercontent.com/7111577/91658831-43e69880-eafe-11ea-8b5c-678f82ba2c86.png)

and something like:
![2020-08-30_21-46](https://user-images.githubusercontent.com/7111577/91660840-c8d7af00-eb0a-11ea-93d1-9aaa422f8431.png)

![2020-09-01_08-09](https://user-images.githubusercontent.com/7111577/91780599-8f966080-ec2a-11ea-834f-82ad16d47939.png)

How could we keep going if the lotus cannot keep up the network ?

**Comment by cmartins88** [comment]
Has anyone managed to overcome this problem, and how? I was still able to synchronize a week ago.

![image](https://user-images.githubusercontent.com/676892/125448978-40562c2b-083f-4266-845e-fd9ddf39af37.png)

lotus version: 1.11.0-rc2 (but I try all since v1.10.0-rc6)
network: calibration

**Comment by Reiers** [comment]
Hi @lionsoul2014 @cmartins88 

Thanks for the report. The issue you are experiencing lies elsewhere.

Moving this issue to Lotus Discussions for help and troubleshooting.

Closing ticket.

**Comment by LexLuthr** [comment, marked as answer]
@lionsoul2014 @cmartins88 Do you still need help with this issue in the latest version?

**Reply by cmartins88** [reply]
All good, thanks. I found that the problem was not with the lotus, but with one of the dependencies, I think with Go...I installed the correct version, started again and everything was back to normal.
